Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.messages.in

```diff
@@ -23,6 +23,10 @@
 
 #if ENABLE(MEDIA_STREAM)
 
+// rdar://140690546 - Can currently be dispatchedTo UI & GPU
+[
+    DispatchedFrom=WebContent
+]
```

Comment:
Should this be removed then? Won't it assert/crash otherwise?

Reply:
We're fine to have a dispatchedFrom here - just not a dispatchedTo (yet).

Reply:
Ah, makes sense.